### PR TITLE
Add docs for options and use custom inspect with "assertSnapshot"

### PR DESCRIPTION
**Pull request by bcheidemann**
Adds docs for https://github.com/denoland/deno_std/pull/2179 and documentation for use of "Deno.customInspect" with "assertSnapshot"

### CODE REVIEW

File: testing/snapshot_testing.md [hunk 1]
```diff
@@ -80,3 +80,317 @@ alongside other code changes. This allows for changes to reference snapshots to
...
+
+The default serialization behaviour can be customised in two ways. The first is
+by specifying the `serializer` option. This allows you to control the
+serialisation of any value which is passed to a specific `assertSnapshot` call.
+See the [above documentation](#options) on the correct usage of this option.
+
+The second option is to make use of `Deno.customInspect`. Because the default
+serializer used by `assertSnapshot` uses `Deno.inspect` under the hood, you can
+set property `Symbol.for("Deno.customInspect")` to a custom serialization
+function.
```
Comment (kt3k) [inline testing/snapshot_testing.md:250]: Nice tip!

**Review by kt3k — APPROVED** [review]
LGTM

Every option is documented very well and `HTMLTag` example looks great! Nice work!

File: testing/snapshot_testing.md (hunk 1 above)
Comment (bcheidemann) [reply testing/snapshot_testing.md:250]: Thanks! ☺️

**Review by bartlomieju — APPROVED** [review]
LGTM too!

**Pull request merged**